cpp/tests/io/parquet_test.cpp

```diff
@@ -5113,4 +5113,58 @@ TEST_P(ParquetSizedTest, DictionaryTest)
   EXPECT_EQ(nbits, GetParam());
 }
 
+TEST_F(ParquetReaderTest, UnsupportedPageEncoding)
+{
+  {
+    // TODO this will need to change when DELTA_BINARY_PACKED is supported
+    // cudf cannot produce a file with a non-supported encoding, so this is a small file
+    // produced by parquet-mr
+    const unsigned char delta_binary_parquet[] = {
+      0x50, 0x41, 0x52, 0x31, 0x15, 0x06, 0x15, 0x1e, 0x15, 0x22, 0x5c, 0x15, 0x06, 0x15, 0x00,
+      0x15, 0x02, 0x15, 0x0a, 0x15, 0x06, 0x15, 0x04, 0x00, 0x00, 0x03, 0x06, 0x03, 0x3f, 0x00,
+      0x0a, 0x24, 0x80, 0x01, 0x04, 0x03, 0x02, 0x02, 0x00, 0x00, 0x00, 0x00, 0x19, 0x11, 0x02,
+      0x19, 0x18, 0x08, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x19, 0x18, 0x08, 0x03,
```

Comment:
I don't know what the current policy is in cuIO, regarding writing out the contents of the file in the test, versus checking in an actual Parquet file. @vuule?

Reply:
If possible, a Python test that open a checked-in file is preferable.

Reply:
:tired_face:

Reply:
Many thanks to @nvdbaranec for python testing tips!